### PR TITLE
FIX: Enable notifications won't stay on #7390

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: android/app/src/main/assets/fiatUnits.json [hunk 1]
```diff
@@ -303,7 +303,7 @@
     "RSD": {
         "endPointKey": "RSD",
         "locale": "sr-RS",
-        "source": "CoinGecko",
+        "source": "CoinDesk",
```
Comment (Overtorment) [inline android/app/src/main/assets/fiatUnits.json:306]: pls try to do separate prs

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!
